content/docs/command-reference/spec.json

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
do we need to keep this file around?

Reply:
technically not for now, I pushed so people can examine it. In future we may or may not keep it, depending on workflow